core/PRP/Region/plVolumeIsect.h

```diff
@@ -75,12 +75,32 @@ class PLASMA_DLL plConvexIsect : public plVolumeIsect
     CREATABLE(plConvexIsect, kConvexIsect, plVolumeIsect)
 
 public:
-    struct PLASMA_DLL SinglePlane
+    class PLASMA_DLL SinglePlane
     {
+    protected:
         hsVector3 fNorm, fPos, fWorldNorm;
         float fDist, fWorldDist;
 
+    public:
         SinglePlane() : fDist(), fWorldDist() { }
+
+        void read(hsStream* S);
+        void write(hsStream* S);
+        void prcWrite(pfPrcHelper* prc);
+        void prcParse(const pfPrcTag* tag);
+
+    public:
+        hsVector3 getNorm() const { return fNorm; }
+        hsVector3 getPos() const { return fPos; }
+        hsVector3 getWorldNorm() const { return fWorldNorm; }
+        float getDist() const { return fDist; }
+        float getWorldDist() const { return fWorldDist; }
+
+        void setNorm(hsVector3 norm) { fNorm = norm; }
+        void setPos(hsVector3 pos) { fPos = pos; }
+        void setWorldNorm(hsVector3 worldNorm) { fWorldNorm = worldNorm; }
```

Comment:
These setters make an extra copy of the `hsVector3` passed in. You can correct this by moving the temporary by eg `fNorm = std::move(norm);`